docs/source/api/user-guide/measurements/randomized-response.rst

```diff
@@ -0,0 +1,323 @@
+        :sync: python
+
+        .. code:: python
+
+            >>> # construct the measurement
+            >>> categories = ["A", "B", "C", "D"]
+            >>> rr_meas = dp.m.make_randomized_response(categories, prob=0.75)
+            
+            >>> # invoke the measurement on some survey response, to execute the randomized response algorithm
+            >>> alice_survey_response = "C"
+            >>> print("noisy release:", rr_meas(alice_survey_response))
+            noisy release: ...
+
+            >>> # determine epsilon by invoking the map
+            >>> print("epsilon:", rr_meas.map(d_in=1))
+            epsilon: 2.197...
+
+Aggregation: Mean
+-----------------
+
+The privatized responses from many individuals may be aggregated to form
+a population-level inference. In the case of the boolean randomized
+response, you may want to estimate the proportion of individuals who
+actually responded with ``True``.
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. code:: python
+
+            >>> import numpy as np
+            >>> num_responses = 1000
+            
+            >>> true_probability = .23
+            
+            >>> private_bool_responses = []
+            
+            >>> for _ in range(num_responses):
+            ...     response = bool(np.random.binomial(n=1, p=true_probability))
+            ...     randomized_response = rr_bool_meas(response)
+            ...     private_bool_responses.append(randomized_response)
+            
+            >>> naive_proportion = np.mean(private_bool_responses)
+            >>> print('naive proportion:', naive_proportion)
+            naive proportion: ...
+
+We know the true probability is .23, so our estimate is off!
+
+The naive proportions can be corrected for bias via the following
+derivation:
+
+.. dropdown:: Derivation of Boolean RR Bias Correction
+
+   We want an unbiased estimate of :math:`\frac{\sum X_i}{n}`. Denote the
+   randomized response :math:`Y_i = \texttt{rr\_bool\_meas}(X_i)`. We first
+   find the expectation of :math:`Y_i`:
+
+   .. math::
+
+      \begin{align*}
+         E[Y_i] &= p X_i + (1 - p) (1 - X_i) \\
+            &= p X_i + p X_i - p - X_i + 1 \\
+            &= (2 p - 1) X_i - p + 1
+      \end{align*}
+
+   This can be used as an unbiased estimator for the proportion of true
+   answers:
+
+   .. math::
+
+      \begin{align*}
+         E[X_i] = \frac{E[Y_i] + p - 1}{2 p - 1}
+      \end{align*}
+
+The resulting expression is distilled into the following function:
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. code:: python
+
+            >>> def debias_randomized_response_bool(mean_release, p):
+            ...     """Adjust for the bias of the mean of a boolean RR dataset."""
+            ...     assert 0 <= mean_release <= 1
+            ...     assert 0 <= p <= 1
+            ...     
+            ...     return (mean_release + p - 1) / (2 * p - 1)
```

Comment:
We should move this into the library proper. Commenting for future issue.

Reply:
- Filed #1806